README.md

```diff
@@ -140,6 +141,30 @@ client = Client.new(url, schema_path: 'config/your_graphql_schema.json')
 client.schema.dump! # you only need to call this when graphql schema changes
 ```
 
+### Preloading schema once
```

Comment:
Capitalize Schema and Once?

Reply:
Makes sense. I should have caught this one!